scripts/API_KEYS.js

```diff
@@ -0,0 +1,3 @@
+module.exports = Object.freeze({
+    type: process.argv.slice(-1)[0]
```

Comment:
I think we can strip out all this `API_KEYS` logic since there's only one relevant API key to handle.

Better to just store it as a secret and reference directly

Reply:
sure, will update this.